docs/ros/installation/robot.mdx

```diff
@@ -56,6 +56,35 @@ sudo apt install ros-humble-clearpath-robot
 
 Installing this package will also install all dependencies required to run any Clearpath robot and supported sensors, other than Clearpath firmware packages.
 
+#### Source install
+
+**1. Create a workspace on the robot's computer**
+
+```
+mkdir ~/clearpath_ws/src -p
+```
+
+**2. Import dependencies**
+
+```
+source /opt/ros/humble/setup.bash
+sudo apt install python3-vcstool
+cd ~/clearpath_ws
+wget https://raw.githubusercontent.com/clearpathrobotics/clearpath_robot/main/dependencies.repos
+vcs import src < dependencies.repos
+rosdep install -r --from-paths src -i -y
+```
+
+**3. Build packages**
+
+```
+colcon build --symlink-install
+```
+
+:::note
+Make sure to add the workspace to your `robot.yaml` file. TODO: Add link to robot yaml documentation
```

Comment:
1. I think there is supposed to be a blank line before and after each ":::" line.
2. Just noting that there is still a "TODO" here.